11-lab/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
```

Comment:
You should use a React.Fragment instead of a div. That way React will remove this wrapper and your rendered page will not have unneeded Div elements.